### ISSUE TITLE
Error at Startup

### ISSUE THREAD

**Issue by KurtKomell** [tooll3/t3#267]
When i start the prog. I throw me an excemption. 
Version 3.61
Grafiktools are installed. .net 6.0 also.
![grafik](https://github.com/tooll3/t3/assets/131547041/e41c6b11-27ab-40e7-93ae-ff747d16164b)

**Comment by pixtur** [comment]
Hi Kurt. Thanks for report.

- Did you run ` StartEditor.exe` ?
- Can you check the size of Operator.dll ?

**Comment by KurtKomell** [comment]
Hello,

yes the operator.dll is 0 kb.

 

Kurt Komell

Fichteplatz 9

07745 Jena

E:  ***@***.***> ***@***.*** M: 0177-4800774

 

Von: Thomas Mann ***@***.***> 
Gesendet: Dienstag, 4. Juli 2023 23:22
An: tooll3/t3 ***@***.***>
Cc: KurtKomell ***@***.***>; Author ***@***.***>
Betreff: Re: [tooll3/t3] Error at Startup (Issue #267)

 

Hi Kurt. Thanks for report.

*	Did you run StartEditor.exe ?
*	Can you check the size of Operator.dll ?

**Comment by KurtKomell** [comment]
Hi, the Operator gets empty when i start the Startt3.exe. I copied again and when I start gets empty again.

 

Kurt Komell

Fichteplatz 9

07745 Jena

E:  ***@***.***> ***@***.*** M: 0177-4800774

 

Von: Thomas Mann ***@***.***> 
Gesendet: Dienstag, 4. Juli 2023 23:22
An: tooll3/t3 ***@***.***>
Cc: KurtKomell ***@***.***>; Author ***@***.***>
Betreff: Re: [tooll3/t3] Error at Startup (Issue #267)

 

Hi Kurt. Thanks for report.

*	Did you run StartEditor.exe ?
*	Can you check the size of Operator.dll ?

**Comment by KurtKomell** [comment]
Hello the problem was copying an old user from the 3.5.1 version into the operators user directory by hand. After I started the unzipped 3.6.1 zip again, the programme worked. I then tried several attempts to import the user via the interface and when I created the same namespace as in the old Tooll3, it worked. Now I can work with it and see what I have tinkered with.

https://youtu.be/6SyaLhNpr60
